package.json

```diff
@@ -153,6 +154,11 @@
         "command": "csharp.listProcess",
         "title": "List process for attach",
         "category": "CSharp"
+      },
+      {
+        "command" : "csharp.listRemoteProcess",
+        "title" : "List processes on remote connection for attach",
+        "category": "CSharp"
```

Comment:
Looking at https://github.com/Microsoft/vscode-node-debug/search?utf8=%E2%9C%93&q=pickNodeProcess, I wonder if there is a way to avoid declaring this command entirely since it shouldn't actually show up in the command well.